design-documents/graph-ql/coverage/custom-attributes/custom-attributes-container.md

```diff
@@ -8,13 +8,42 @@ One workaround for "getting all fields" is based on schema introspection, it all
 
 # Proposed solution
 
-To account for dynamic nature of EAV attributes and the need of "getting all fields" in product search queries, `custom_attributes: [CustomAttribute]!` container will be introduced. 
+To account for dynamic nature of EAV attributes and the need of "getting all fields" in product search queries, we can introduce `custom_attributes: [CustomAttribute]!` container (recommended approach). 
```

Comment:
do fragments bring any value for PWA since we still prefer these flat single structure arrays vs infterfaces?